ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -133,6 +133,14 @@
   @Default("1")
   public final int storeMinLogSegmentCountToReclaimToTriggerCompaction;
 
+  /**
+   * Comma separated list of the compaction triggers that should be enabled. The valid triggers are:
+   * Time, Admin
+   */
+  @Config("store.compaction.triggers")
```

Comment:
So the default for the triggers should be an empty string? Because the default for enable compaction is `false`